bvm/ballerina-runtime/src/test/java/io/ballerina/runtime/test/config/TomlProviderTest.java

```diff
@@ -634,13 +631,6 @@ public void testTomlProviderUnions(String variableName, Type type, Object expect
                                                                 ValueCreator.createKeyFieldEntry(fromString("age"),
                                                                                                  14L),
                                                         })},
-                // union variable with array value
-                {"arrayUnionVar", TypeCreator.createUnionType(List.of(TYPE_INT, arrayType), true),
```

Comment:
Any reason why we remove union test?

Reply:
This test checks the returning Array value with the expected Array value (which are two different objects). Since we removed the `equals` method for the fix, this will fail. Therefore, I moved this test to the configurable integration test.